docs/app/Examples/views/Card/Variations/Colored.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { Card, Image } from 'stardust'
+
+const ColoredCards = () => (
+  <div>
+    <Card.Group itemsPerRow={4}>
+      <Card color='red'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='orange'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='yellow'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='olive'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='green'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='teal'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='blue'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='violet'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='purple'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='pink'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='brown'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+      <Card color='grey'>
+        <Image src='http://semantic-ui.com/images/wireframe/white-image.png' />
+      </Card>
+    </Card.Group>
+  </div>
```

Comment:
I think the `Card` should take an `image` shorthand prop.  This will make it more consistent with other component shorthand.  Also, it reduces this markup considerably:

```jsx
import React from 'react'
import { Card } from 'stardust'

const src = 'http://semantic-ui.com/images/wireframe/white-image.png'

const ColoredCards = () => (
  <Card.Group itemsPerRow={4}>
    <Card color='red' image={src} />
    <Card color='orange' image={src} />
    <Card color='yellow' image={src} />
    <Card color='olive' image={src} />
    <Card color='green' image={src} />
    <Card color='teal' image={src} />
    <Card color='blue' image={src} />
    <Card color='violet' image={src} />
    <Card color='purple' image={src} />
    <Card color='pink' image={src} />
    <Card color='brown' image={src} />
    <Card color='grey' image={src} />
  </Card.Group>
)

export default ColoredCards
```